### PR TITLE
Review expectations for software engineers conveying to candidates

### PR DESCRIPTION
**Pull request by garryyao**
After discussed in #guild-hiring we're updating the expectations of the tech task and being more explicit about them, also get rid of some outdated contents.
 - what's are we looking for
 - how to submit
 - improve option2

### CODE REVIEW

**Comment by garryyao** [comment]
Close for an internal discussion.

File: README.md [hunk 1]
```diff
@@ -1,69 +1,28 @@
...
+
+This is a coding exercise to help us assess candidates looking to join the team at Condé Nast International.  The test is intended to be completed at home, and we expect you to spend no more than half a day working it out. We appreciate the time you spent on it and will having it examined by one of our engineer, we will go through your submitted work together when you're invited to meet us face-to-face.        
+
+## Which test shall I pick?
+ - [Software Engineer](se-test)
+ - [Infrastructure Engineer](infra-test)   
+
+## How to submit code
+We prefer to see your work as a GIT repo with commits from which the app is built up.
+You can submit by sending up the link to a public git repository on Github, or if for some reason you'd
```
Comment (leeDav) [inline README.md]:
```suggestion
You can submit by sending up the link to a public git repository on Github, or if you'd 
```
The "if for some reason" could come across passive-aggressive, or at least implies we would think negatively of you if you provided us with this.

I know this was not your intent, though! Thoughts?

File: README.md [hunk 2]
```diff
@@ -1,69 +1,28 @@
...
+Send the work to your **point of contact**. 
+
+## What we are looking for?
+These points serve as a guide on what we look for in a code submission. Not all points have to be met
+ - The solution fulfils the brief
+ - There is some attempt at creating documentation
+ - There is a README that instructs how to run the app
+ - The app runs when you follow the README or `npm start` (or equivalent)
+ - There are reasonably thorough tests
+ - The tests were written/committed before the code was
```
Comment (leeDav) [inline README.md]: I don't think we need to go in to this much detail, especially about the tests being written before the code was.

File: se-test/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Given the fact that this is a Full Stack challenge, we are expecting a Frontend application and a Backend application that communicate with each other.
+
+A Full Stack developer is someone with familiarity in each layer; you might be stronger on the frontend, or vice-versa. Feel free to focus more on what you know best, be creative and show us your skills!
+
+### Submission
+See ["how to submit your work"](../README.md#how-to-submit-code)
+
+### FAQ
+
+Q: Shall I provide my `newsapi.org` token?
```
Comment (leeDav) [inline se-test/README.md]:
```suggestion
Q: Should I commit my `newsapi.org` token?  
```

File: se-test/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+A Full Stack developer is someone with familiarity in each layer; you might be stronger on the frontend, or vice-versa. Feel free to focus more on what you know best, be creative and show us your skills!
+
+### Submission
+See ["how to submit your work"](../README.md#how-to-submit-code)
+
+### FAQ
+
+Q: Shall I provide my `newsapi.org` token?  
+A: Absolutely not. Provide a way we can easily plug it in
```
Comment (leeDav) [inline se-test/README.md]:
```suggestion
A: No, but please provide one for us
```
`Absolutely not` sounds quite aggressive. As above, I know this isn't your intent.

File: se-test/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Q: What browsers must I support?  
+A: Latest Firefox & Chrome. Don’t worry about any browser inconsistencies.
+
+Q: What version of node must I support?
```
Comment (leeDav) [inline se-test/README.md]:
```suggestion
Q: What version of Node must I support?  
```

File: se-test/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+A: >= 8
+
+Q: Should I deploy the application somewhere it is publicly reachable?
+A: No
+
+Q: Can I use transpilers?  
+A: Yes, but you [may not need to](http://kangax.github.io/compat-table/es6/).
+
+Q: Can I use libraries, frameworks, etc?  
+A: Yes...but please consider ["What we are looking for?"](../README.md#What we are looking for?) guidelines.
```
Comment (leeDav) [inline se-test/README.md]:
```suggestion
A: Yes... but please consider ["What we are looking for?"](../README.md#What we are looking for?) guidelines.
```

**Pull request merged**